lib/node/nodes/data-observatory-multiple-measures.js

```diff
@@ -123,7 +125,7 @@ var queryTemplate = Node.template([
     '_data AS ( ',
     '  SELECT id AS __obs_id__, {{=it.obsColumns}}',
     '  FROM cdb_dataservices_client._OBS_GetData_exception_safe(',
-    '    (SELECT ARRAY_AGG((the_geom, cartodb_id)::geomval) FROM _source),',
+    '    (SELECT geomvals FROM _summary),',
```

Comment:
That was the intention I guess. I moved it there to avoid duplicate the WHERE clause and made it error prone to future changes. It's the same query, more or less, we have in the old node.

I'd really appreciate if you say what potential problem we're facing to fix it and publish a new version asap. If is just about the naming I'll create an issue to have it in mind for a future version instead of publishing a new one.